### PR TITLE
e2e: fix long api groups exhausting label value length

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

File: test/e2e/framework/util.go [hunk 1]
```diff
@@ -345,3 +347,14 @@ func ConfigWithToken(token string, cfg *rest.Config) *rest.Config {
 	cfgCopy.BearerToken = token
 	return cfgCopy
 }
+
+// UniqueGroup returns a unique API group with the given suffix by prefixing
+// some random 8 character base62 string. suffix must start with a dot if the
```
Comment (ncdc) [inline test/e2e/framework/util.go:352]:
```suggestion
// some random 8 character base36 string. suffix must start with a dot if the
```

Comment (sttts) [reply test/e2e/framework/util.go]: updated

**Review by ncdc — APPROVED** [review]

**Comment by ncdc** [comment]
```
error creating CRD sheriffs.aGL5FUSX.io: CustomResourceDefinition.apiextensions.k8s.io "sheriffs.aGL5FUSX.io" is invalid: [metadata.name: Invalid value: "sheriffs.aGL5FUSX.io": a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*'), spec.group: Invalid value: "aGL5FUSX.io": a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')]
```

**Comment by sttts** [comment]
> `sheriffs.aGL5FUSX.io`

didn't you say I do base36, not base62? :)

**Comment by ncdc** [comment]
Just needs to be lowercase, right?

**Comment by sttts** [comment]
looks like.

**Comment by sttts** [comment]
pushed

**Review by ncdc — APPROVED** [review]

**Pull request merged**